### PR TITLE
RICT-3770 Be able to bundle a wrapper app

### PR DESCRIPTION
**Pull request by alvesmiguel1**

### CODE REVIEW

**Comment by alvesmiguel1** [comment]
So, this will be the process:
- Inner app is bundled (like before)
- Wrapper app is bundled next. Some properties are able to be overriden.
- Files are put in place in a script running after this.

**Review by joaompneves — CHANGES_REQUESTED** [review]
I dont like the fact that you brought the wrapper concept here.... This is a public project that we should try to maintain as simple and generic as possible. This way it will be harder to create PRs to be accepted in the original fork

**Comment by alvesmiguel1** [comment]
I don't like it as well but I don't foresee a better possibility here... But I'll try to override properties via msbuild command-line instead and remove "wrapper" stuff

File: DotNet.Bundle/build/DotNet.Bundle.targets [hunk 1]
```diff
@@ -33,15 +32,26 @@
...
+      <CFBundleURLTypesValues>@(CFBundleURLTypes)</CFBundleURLTypesValues>
 
       <NSPrincipalClass Condition="'$(NSPrincipalClass)' == ''">NSApplication</NSPrincipalClass>
       
-      <NSHighResolutionCapable Condition="'$(NSHighResolutionCapable)' == ''" >true</NSHighResolutionCapable>
-      
-    </PropertyGroup>
+      <CFBundleURLTypesValues>@(CFBundleURLTypes)</CFBundleURLTypesValues>
     
+      <LSEnvironmentValues>@(LSEnvironemnt)</LSEnvironmentValues>
```
Comment (joaompneves) [inline DotNet.Bundle/build/DotNet.Bundle.targets]: typo

**Review by joaompneves — CHANGES_REQUESTED** [review]

File: DotNet.Bundle/build/DotNet.Bundle.targets (hunk 1 above)
Comment (alvesmiguel1) [reply DotNet.Bundle/build/DotNet.Bundle.targets]: Fixed. Had to slightly change how we handle ItemGroups (ITaskItem)

**Review by joaompneves — APPROVED** [review]

**Review by plemasantos — APPROVED** [review]

**Pull request merged**